includes/Generator/Plugins/SchemaOrg.php

```diff
@@ -153,17 +170,7 @@ private function getImageMetadata() {
 			}
 		}
 
-		try {
-			$logo = MediaWikiServices::getInstance()->getMainConfig()->get( 'Logo' );
-			$logo = wfExpandUrl( $logo );
-			$data['url'] = $logo;
-		} catch ( Exception $e ) {
-			// Uh oh either there was a ConfigException or there was an error expanding the URL.
-			// We'll bail out.
-			$data = [];
-		}
-
-		return $data;
+		return $image;
```

Comment:
This will cause an undefined notice if no image is set.  
I'd like to have the use of the sites logo as a fallback reintroduced here :)